src/Sylius/Bundle/ApiBundle/spec/Doctrine/QueryCollectionExtension/FilterEagerLoadingExtensionSpec.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\ApiBundle\Doctrine\QueryCollectionExtension;
+
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Extension\ContextAwareQueryCollectionExtensionInterface;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Util\QueryNameGeneratorInterface;
+use Doctrine\ORM\QueryBuilder;
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\Order;
+use Sylius\Component\Core\Model\Product;
+
+final class FilterEagerLoadingExtensionSpec extends ObjectBehavior
+{
+    function let(ContextAwareQueryCollectionExtensionInterface $decoratedExtension): void
+    {
+        $this->beConstructedWith(
+            $decoratedExtension,
+            ['shop_get' => ['resources' => [Product::class => ['enabled' => true]]]]
```

Comment:
Minor thing, but we don't have a test for the "false" value under the enabled key